token-lending/program/src/instruction.rs

```diff
@@ -914,3 +952,41 @@ pub fn set_lending_market_owner(
         data: LendingInstruction::SetLendingMarketOwner { new_owner }.pack(),
     }
 }
+
+/// Creates a `FlashLoan` instruction.
+#[allow(clippy::too_many_arguments)]
+pub fn flash_loan(
+    program_id: Pubkey,
+    amount: u64,
+    reserve_liquidity_pubkey: Pubkey,
+    destination_liquidity_pubkey: Pubkey,
+    reserve_pubkey: Pubkey,
+    lending_market_pubkey: Pubkey,
+    derived_lending_market_authority: Pubkey,
+    flash_loan_receiver_pubkey: Pubkey,
+    flash_loan_fees_account_info: Pubkey,
+    host_fee_recipient: Pubkey,
+    flash_loan_receiver_program_params: Vec<Pubkey>,
```

Comment:
Naming should be consistent across the board. Everything else is a `_pubkey`

Reply:
fixed.